src/modules/roles/application/use-cases/__tests__/update-user-roles.use-case.spec.ts

```
@@ -99,7 +114,12 @@
         roles: [remainingRole],
       });
 
-      const result = await useCase.execute(userId, roleId, undefined, currentUserId);
+      const result = await useCase.execute(
```

ESLint

[error] 'result' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)